### ISSUE TITLE
LFS upload failed

### ISSUE THREAD

**Issue by gwillig** [git-lfs/git-lfs#3674]
I try to push a local file to my git repo. But the lfs upload files.
It tells me that the file is missing. But file is there, I can see it on my computer 

That is the repo: https://github.com/gwillig/uk_accident_analysis

1) Attempt:  
 git init  
git remote add origin https://github.com/gwillig/UK-traffic-accidents.git  
git lfs install  
git track "*.csv"  
git commit -m "first commit"  
git push origin master  

=> Result: Same error message as in the screenshot

2) Attempt:  
 git config -f .lfsconfig lfs.url https://github.com/gwillig/UK-traffic-accidents.git  

Error message
[![enter image description here][1]][1]

Folder:

[![enter image description here][2]][2]

  [1]: https://i.stack.imgur.com/P1hxT.png
  [2]: https://i.stack.imgur.com/MJWsq.png

**Comment by bk2204** [comment]
Hey, thanks for writing in, and sorry you're having trouble.

It looks like you're missing the object for the file you checked in. Does `git lfs fsck` indicate that everything's okay, or does it show the file as missing? If it shows it as missing, are there other error messages that you get when doing `git add` or `git commit`? Also, what does running `find .git/lfs/objects/ -type f` show?

**Comment by gwillig** [comment]
Hello bk2204,

thanks for your response, with your help I found the error. There where two erros.
1.) My user name has a space 

2.) The configuration of the endpoint was wrong: 
It was 
`git lfs env` I found out that the configuration of my endpoint was wrong. 
It was  https://github.com/gwillig/uk_accident_analysis.git. The right endpoint is   git config lfs.url "https://github.com/gwillig/uk_accident_analysis.git/info/lfs"
